cmd/juju/commands/bootstrap.go

```diff
@@ -204,12 +208,42 @@ type bootstrapCommand struct {
 }
 
 func (c *bootstrapCommand) Info() *cmd.Info {
-	return jujucmd.Info(&cmd.Info{
+	info := &cmd.Info{
 		Name:    "bootstrap",
 		Args:    "[<cloud name>[/region] [<controller name>]]",
 		Purpose: usageBootstrapSummary,
-		Doc:     usageBootstrapDetails,
-	})
+	}
+	if details := c.configDetails(); len(details) > 0 {
+		if output, err := common.FormatConfigSchema(details); err == nil {
+			info.Doc = fmt.Sprintf("%s%s\n%s%s",
+				usageBootstrapDetailsPartOne,
+				usageBootstrapConfigTxt,
+				output,
+				usageBootstrapDetailsPartTwo)
+			return jujucmd.Info(info)
+		}
+	}
+	info.Doc = strings.TrimSpace(fmt.Sprintf("%s%s",
+		usageBootstrapDetailsPartOne,
+		usageBootstrapDetailsPartTwo))
+
+	return jujucmd.Info(info)
+}
+
+func (c *bootstrapCommand) configDetails() map[string]interface{} {
+	result := map[string]interface{}{}
+	addAll := func(m map[string]interface{}) {
+		for k, v := range m {
+			result[k] = v
```

Comment:
Do we want to check that we have overwritten anything? I guess we control the keys so not really a problem.

Reply:
Not really... Also I believe that controller config would take precedence anyway ; )